src/signals/incident-management/containers/IncidentDetail/components/AttachmentViewer/index.test.js

```diff
@@ -1,10 +1,107 @@
-// import React from 'react';
-// import { shallow } from 'enzyme';
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
 
-// import ImageViewer from 'index';
+import AttachmentViewer from './index';
 
-describe('<ImageViewer />', () => {
-  it('Expect to have unit tests specified', () => {
-    expect(true).toEqual(true);
+describe('<AttachmentViewer />', () => {
+  let props;
```

Comment:
Why is `props` a `let`? Its value doesn't change.

Reply:
gedaan